src/zimscraperlib/imaging.py

```diff
@@ -40,8 +40,27 @@ def solarize(r, g, b):
     return rgb_to_hex(mr, mg, mb), rgb_to_hex(sr, sg, sb)
 
 
+def alpha_not_supported():
```

Comment:
Is there a reason for doing it as a function instead of a global variable or is it just a preference of yours?

Reply:
I just did that as Codefactor doesn't like globals generally. And we have an A+ rating on this repo.

Reply:
It's updating global variable that is problematic as it leads to uncontrollable situations.

Reply:
Okay. Made it a global var